query/stdlib/testing/testing.go

```diff
@@ -50,6 +50,7 @@ var FluxEndToEndSkipList = map[string]string{
 	"integral":         "unbounded test",
 	"integral_columns": "unbounded test",
 	"map":              "unbounded test",
+	"buckets":          "unbounded test",
```

Comment:
This is a new test that was added that didn't have a `range` call?

Reply:
O sorry, I saw it now, yes